front/app/containers/Admin/sideBar/messages.ts

```diff
@@ -57,12 +57,20 @@ export default defineMessages({
     id: 'app.containers.AdminPage.SideBar.insights',
     defaultMessage: 'Reporting',
   },
+  academy: {
+    id: 'app.containers.AdminPage.SideBar.academy',
+    defaultMessage: 'Academy',
+  },
+  linkToAcademy: {
+    id: 'app.containers.AdminPage.SideBar.linkToAcademy',
+    defaultMessage: 'https://academy.citizenlab.co/',
+  },
   guide: {
     id: 'app.containers.AdminPage.SideBar.guide',
     defaultMessage: 'Guide',
   },
-  linkToSupportCenter: {
-    id: 'app.containers.AdminPage.SideBar.linkToSupportCenter',
-    defaultMessage: 'https://support.citizenlab.co/',
+  linkToGuide: {
+    id: 'app.containers.AdminPage.SideBar.linkToGuide',
+    defaultMessage: 'https://community.citizenlab.co/en/folders/guide',
```

Comment:
I'm not sure about this one either way. If we have multiple versions of this page (doesn't seem like we do yet, I checked FR and DE and they redirected to EN) then we might want to just swap the locale in JS rather than have the translators do it, and if we don't have translations for the page I think we can just leave the link in plain text in the UI (untranslated)

Reply:
German version is here https://community.citizenlab.co/de-DE/folders/guide

First, I also thought the values in the code would be fine, but then I realized the rules to change the URL (to add locale name to it) can be tricky and can change in the future. See the comments here https://citizenlab.atlassian.net/browse/CL-1088

Reply:
Cool, that makes sense 👍